apps/builder/app/shared/nano-states/instances.ts

```diff
@@ -2,13 +2,18 @@ import { atom, computed } from "nanostores";
 import type { Instance, Instances } from "@webstudio-is/project-build";
 import type { InstanceSelector } from "../tree-utils";
 import { useSyncInitializeOnce } from "../hook-utils";
+import type { ItemId } from "node_modules/@webstudio-is/design-system/src/components/tree/item-utils";
 
 export const isResizingCanvasStore = atom(false);
 
 export const selectedInstanceSelectorStore = atom<undefined | InstanceSelector>(
   undefined
 );
 
+export const editInstanceNameSelectorStore = atom<undefined | ItemId>(
```

Comment:
why is it called selector? When we say selector in instance case, its an array that represents a tree path. in this case its just an id, no?

Reply:
I was trying to get used to the state management stuff. I kinda copied the above behaviour 😅. Will update the names 👍